group_vars/location_rhnk/networks.yml

```diff
@@ -43,6 +43,8 @@ networks:
     name: mesh_fhain
     prefix: 10.230.3.17/32
     ipv6_subprefix: -17
+    mesh_metric_lqm:
+    - default 0.2 # make sure emma is uplink and not f2a
```

Comment:
That interface is called mesh_fhain. Is f2a the only link? Otherwise please only apply that lqm for the peer ip.

Reply:
I am not completely sure what is the plan. I guess if it is no ptp link it would be called, `mesh_n` or something like that. However, maybe @a22sc or @booo , or @Koltonowski. I updatet that now to only apply the penalty to f2a.